app/src/main/kotlin/com/wire/android/ui/home/conversations/search/InternalContactSearchResultItem.kt

```diff
@@ -142,7 +142,7 @@ fun ExternalContactSearchResultItem(
         },
         actions = {
             when (connectionState) {
-                ConnectionState.NOT_CONNECTED ->
+                ConnectionState.NOT_CONNECTED, ConnectionState.CANCELLED ->
```

Comment:
Is this change on the scope of this PR?

Reply:
Maybe this change is related to your other ticket AR-2086 and got here by accident? 🤔

Reply:
Yeah, I saw some error messages and by mistake added it here, can it stay?